linux-glibc-devel.spec

```diff
@@ -1,13 +1,13 @@
 Summary: Headers describing the kernel ABI
-Name: kernel-headers
-Group: System/Kernel
+Name: linux-glibc-devel
 License: GPLv2
 URL: http://www.kernel.org/
 
 %define kversion 3.18.136
 Version: %{kversion}
 Release: 1
 Provides: kernel-headers = %{kversion}
+Obsoletes: kernel-headers <= %{kversion}
```

Comment:
I think this should be lower the current version not lower equal.

Reply:
Because of the package type and the version used in the dependency it has to be lower equal to replace the current version. It can be changed if packaging type is changed and the version is changed to something that changes with each package version.